nvOC

```diff
@@ -210,16 +210,51 @@ Reboot()
    sudo su -c "shutdown -r now"
 }
 
+
+############################### Reboot host ###################################
+Reboot()
+{
+   # Reboots the host
+   echo -e "${B}${R}Warning: The host will reboot in 10 seconds, ctrl-c to cancel...${N}"
+   sleep 10
+   sudo su -c "shutdown -r now"
+}
+############################### Reboot host ###################################
+Restart()
+{
+   # Reboots the host
+        nvoc stop && timeout 2 && nvoc start
+}
 ############################### Start nvOC ####################################
 Start()
 {
-    if [ $LOCALorREMOTE == "LOCAL" ];then
-       gnome-terminal
-    else   
-       Reboot
-    fi   
+   if [ $LOCALorREMOTE == "LOCAL" ];then
+   pkill -f gnome-terminal
+   pkill -f 5watchdog
+   pkill -f 7telegram
+   pkill -f 6tempcontrol
+   pkill -e screen
+   pkill -f 8wtm
+      gnome-terminal -e "bash '/home/m1/3main'"
+    echo -e "${B}Miner Restart....${N}"; echo ""
+      fi
+
+   if [ $LOCALorREMOTE == "REMOTE" ];then
+        echo -e "${B}Stopping nvOC processes...Ghost${N}"; echo ""
+        # Kill off the nv_OC processes
+   pkill -f gnome-terminal
+   pkill -f 5watchdog
+   pkill -f 7telegram
+   pkill -f 6tempcontrol
+   pkill -e screen
+   pkill -f 8wtm
+     export DISPLAY=:0
+    gnome-terminal -e "bash '/home/m1/3main'"
```

Comment:
Same thing here, but I'd rather remove those two duplicate IF statements and only perform the `export DISPLAY=:0` in case of REMOTE.